.env

```diff
@@ -1 +1 @@
-REACT_APP_BACKEND_ADDRESS=http://localhost
\ No newline at end of file
+REACT_APP_BACKEND_ADDRESS=http://localhost:5000
```

Comment:
The backend server runs on port 80 when started with Docker, which is how the backend server will be running on the Pis. Since port 80 is the default web port, it can be ommitted.